platforms/r3-corda-ent/charts/cenm-gateway/templates/job.yaml

```diff
@@ -8,128 +8,108 @@
-        - name: {{ .Chart.Name }}-check-auth
-          securityContext:
-            {{- toYaml .Values.config.securityContext | nindent 12 }}
-          image: "{{ .Values.image.gatewayContainerName }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+            echo "Connection gateway.{{ .Release.Namespace }} port {{ .Values.global.cenm.gateway.port }} successful"
+        - name: init-check-auth
+          image: {{ .Values.image.gateway.repository }}:{{ .Values.image.gateway.tag }}
+          imagePullPolicy: IfNotPresent
           command: ["/bin/bash", "-c"]
           args:
           - |-
-            echo Probing {{ .Values.cenmServices.authName }}.{{ .Values.metadata.namespace }} port {{ .Values.cenmServices.authPort }}
-            until nc -w 5 -vz {{ .Values.cenmServices.authName }}.{{ .Values.metadata.namespace }} {{ .Values.cenmServices.authPort }} > /dev/null 2>&1
+            echo Probing  auth.{{ .Release.Namespace }} port {{ .Values.global.cenm.auth.port }}
+            until nc -w 5 -vz auth.{{ .Release.Namespace }} {{ .Values.global.cenm.auth.port }} > /dev/null 2>&1
             do
-              echo "Connect to {{ .Values.cenmServices.authName }}.{{ .Values.metadata.namespace }} port {{ .Values.cenmServices.authPort }} (tcp) failed: Connection refused"
+              echo "Connect to auth.{{ .Release.Namespace }} port {{ .Values.global.cenm.auth.port }} (tcp) failed: Connection refused"
             done
-            echo "Connection {{ .Values.cenmServices.authName }}.{{ .Values.metadata.namespace }} port {{ .Values.cenmServices.authPort }} successful"
-          resources:
-            {{- toYaml .Values.config.pod.resources | nindent 12 }}
+            echo "Connection auth.{{ .Release.Namespace }} port {{ .Values.global.cenm.auth.port }} successful"
       containers:
         - name: main
-          securityContext:
-            {{- toYaml .Values.config.securityContext | nindent 12 }}
-          image: "{{ .Values.image.gatewayContainerName }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: {{ .Values.image.gateway.repository }}:{{ .Values.image.gateway.tag }}
+          imagePullPolicy: IfNotPresent
           command: ["/bin/bash", "-c"]
           args:
           - |-
-            pwd
-            cd {{ .Values.config.volume.baseDir }}/CM-FILES/
-            ls -alR
-            ./setupAuth.sh {{ .Values.nodeName }}.{{ .Values.metadata.namespace }} {{ .Values.cenmServices.gatewayPort }}
-          resources:
-            {{- toYaml .Values.config.pod.resources | nindent 12 }}
+            # run the auth users setup
+            cd {{ .Values.baseDir }}/CM-FILES
+            ./setupAuth.sh gateway.{{ .Release.Namespace }} {{ .Values.global.cenm.gateway.port }}
           volumeMounts:
           - name: gateway-conf
-            mountPath: {{ .Values.config.volume.baseDir }}/CM-FILES/setupAuth.sh
+            mountPath: {{ .Values.baseDir }}/CM-FILES/setupAuth.sh
```

Comment:
I think Values,baseDir can just be replaced with /opt/corda or whatever we are using rather than it being a variable

Reply:
ok sure will update

Reply:
done